llm-monitoring-main.ipynb

```diff
@@ -62,36 +62,21 @@
    "execution_count": null,
```

Comment:
<p>Avoid using general and short variable names. <code>func</code> -&gt; <code>training_function</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demo-monitoring-and-feedback-loop/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Reply:
<p>I remove those lines, they are not mandatory and by mistake they were in the notebook </p>